policyengine_us/reforms/states/dc/dc_ctc.py

```diff
@@ -59,9 +60,18 @@ def create_dc_ctc_reform(parameters, period, bypass: bool = False):
     if bypass:
         return create_dc_ctc()
 
-    p = parameters(period).gov.contrib.states.dc.ctc
+    p = parameters.gov.contrib.states.dc.ctc
 
-    if p.in_effect:
+    reform_active = False
+    current_period = period_(period)
+
+    for i in range(5):
+        if p(current_period).in_effect:
+            reform_active = True
```

Comment:
can replace all the below
```suggestion
            return create_dc_ctc()
        current_period = current_period.offset(1, "year")
    return None
```